pkg/oracle/transaction.go

```diff
@@ -61,13 +80,14 @@ func (t *incompleteTx) finalize(oracleNodes keys.PublicKeys) bool {
 		}
 	}
 	if len(sigs) != m {
-		return false
+		failed := len(t.sigs)-len(sigs) > len(oracleNodes)-m
```

Comment:
How would you ever get `true` here if you're deleting or not accepting wrong signatures anyway?